src/index.tsx

```diff
@@ -115,7 +133,7 @@ var loadHere = (): boolean => {
 }
 
 var getRandomUrlFromFolder = (): string => {
-    return 'https://random.url.com/';
+    return 'https://example.com/random';
```

Comment:
Although I don't mind it being changed, I do believe it's less readable.

Reply:
url.com belongs to [Next Small Things](http://tnst.de/) , and previous url resolved to 503 error. Because this is a placeholder anyways, why not use example.com instead?

Reply:
Valid point. 👍